researchstudio-sat/webofneeds
# Feat multiselect msgs

First Implementation of multi-selection "actions":
Adds the possibility to retract/reject/propose/accept/proposeToCancel... multiple messages at once

How to test:
1) chat with debug bot
2) open the + next to the chattextfield -> see the new action buttons
3) try any of them and "execute" the action in the footer (you can combine as you see fit and even add details)
4) show AgreementData -> open + menu -> see another batch of buttons -> rinse and repeat

fixes #2270
